client/src/pages/create-database.tsx

```diff
@@ -104,118 +130,207 @@ export const CreateDatabase = () => {
 
   const isPluginInstalled = data?.isPluginInstalled.isPluginInstalled;
 
+  const retrieveDbId = (logs: RealTimeLog[]) => {
+    const lastLog = logs[logs.length - 1];
+    const dbId = lastLog.message?.substring(
+      lastLog.message.indexOf(':') + 2,
+      lastLog.message.length
+    );
+    return dbId;
+  };
+
+  const handleNext = () => {
+    setIsTerminalVisible(false);
+    const dbId = retrieveDbId(arrayOfCreateDbLogs);
+    history.push(`database/${dbId}`);
+  };
+
   useEffect(() => {
     isDokkuPluginInstalled({
       variables: {
         pluginName: dbTypeToDokkuPlugin(formik.values.type),
       },
     });
-  }, [formik.values.type, isPluginInstalled, isDokkuPluginInstalled]);
+    isDbCreationSuccess === DbCreationStatus.FAILURE
+      ? toast.error('Failed to create database')
+      : isDbCreationSuccess === DbCreationStatus.SUCCESS &&
+        toast.success('Database created successfully');
+  }, [
+    formik.values.type,
+    isPluginInstalled,
+    isDokkuPluginInstalled,
+    isDbCreationSuccess,
+  ]);
 
   return (
     <React.Fragment>
       <Header />
 
       <div className="max-w-5xl mx-auto py-6 px-4 sm:px-6 lg:px-8">
         <h1 className="text-lg font-bold">Create a new database</h1>
+        <div className="mt-12">
+          {isTerminalVisible ? (
+            <React.Fragment>
+              <p className="mb-2 ">
+                Creating <b>{formik.values.type}</b> database{' '}
+                <b>{formik.values.name}</b>
+              </p>
+              <Terminal className={'w-6/6'}>
+                <p className="text-green-400 mb-2">
```

Comment:
in my personal opinion text inside the terminal should come from the server only. What do you think about moving this sentence in top of the terminal? Maybe in some light grey color?